### ISSUE TITLE
[Client Step 5.3]

### ISSUE THREAD

**Issue by braedongough** [Urigo/WhatsApp-Clone-Tutorial#14]
I'm getting an error when using fetch in the test: 

```
Property 'mockResponseOnce' does not exist on type '{ (input: RequestInfo, init?: RequestInit | undefined): Promise<Response>; (input: RequestInfo, init?: RequestInit | undefined): Promise<Response>; }'.ts(2339)
```

This is resolved by using fetchMock instead - should this be updated in the tutorial?

**Comment by Urigo** [comment]
mmm can you create an example PR to what you think needs to be changed?

**Comment by braedongough** [comment]
Line 10 in `src/components/ChatsListScreen/ChatsList.test.tsx` on branch master-step5, 

`fetch.mockResponseOnce(` -> `fetchMock.mockResponseOnce(`

**Comment by dgdosen** [comment]
I get this error:

    TypeError: fetch.mockResponseOnce is not a function

**Comment by Urigo** [comment]
it makes sense but I can't understand why I don't get this error myself..

What are you running when you get it?
If you clone the repo from that exact point, do you still get that error?

**Comment by Urigo** [comment]
@braedongough you are right, it was broken.
I've fixed it (also in step 6.4) and deployed a new version and added you to the contributors on the bottom of the README
